store/postgres/src/primary.rs

```diff
@@ -322,37 +410,71 @@ impl<'a> Connection<'a> {
         }
     }
 
+    /// Signal any copy process that might be copying into one of these
+    /// deployments that it should stop. Copying is cancelled whenever we
+    /// remove the assignment for a deployment
+    fn cancel_copies(&self, ids: Vec<DeploymentId>) -> Result<(), StoreError> {
+        use active_copies as ac;
+
+        update(ac::table.filter(ac::dst.eq_any(ids)))
+            .set(ac::cancelled_at.eq(sql("now()")))
+            .execute(self.0.as_ref())?;
+        Ok(())
+    }
```

Comment:
It seems worth logging that a copy was canceled.

Reply:
Setting this column only requests cancellation. The copy process will check this periodically, and then stop. When it stops, it actually logs a warning that copying was cancelled and is incomplete (in `copy::copy_data`)